### DISCUSSION TITLE
PCL2 下载能否用一个 Github repo 放在 release 里？

### DISCUSSION THREAD

**Discussion by amorphobia** [LTCatt/PCL2Help#209] [General]
感谢你的精彩作品！

我自己维护了一个 [Scoop](https://scoop.sh) 的库 [siku](https://github.com/amorphobia/siku)，收录了 [PCL2](https://github.com/amorphobia/siku/blob/master/bucket/plain-craft-launcher.json)。但官方下载链接都是网盘链接，无法在命令行下载，也不方便用 Github action 自动更新。

目前我使用了 [hanximeng/LanzouAPI](https://github.com/hanximeng/LanzouAPI) 提供的示例地址解析蓝奏云地址，但这也不是长久的方式，因此希望官方可以提供 Github release 的下载方式。

再次感谢！

**Comment by Big-Cake-jpg** [comment]
本 Repository 仅限 PCL2 帮助库相关讨论。

你应该前往 爱发电 找龙猫私聊

**Comment by z0z0r4** [comment]
> 感谢你的精彩作品！
> 
> 我自己维护了一个 [Scoop](https://scoop.sh) 的库 [siku](https://github.com/amorphobia/siku)，收录了 [PCL2](https://github.com/amorphobia/siku/blob/master/bucket/plain-craft-launcher.json)。但官方下载链接都是网盘链接，无法在命令行下载，也不方便用 Github action 自动更新。
> 
> 目前我使用了 [hanximeng/LanzouAPI](https://github.com/hanximeng/LanzouAPI) 提供的示例地址解析蓝奏云地址，但这也不是长久的方式，因此希望官方可以提供 Github release 的下载方式。
> 
> 再次感谢！

似乎即将开源？未来可能有吧
